setup.py

```diff
@@ -34,17 +34,24 @@
 # ============================================================================
 #
 import setuptools
+from os.path import join, dirname, isfile
 
 with open("README.md", "r") as file:
-	long_description = file.read()
+    long_description = file.read()
+
+projectName = "pyVHDLParser"
+
+rfile = "requirements.txt"
+if not isfile(rfile):
+	rfile = join(dirname(__file__), projectName + ".egg-info", "requires.txt")

```

Comment:
Please use [`Pathlib`](https://docs.python.org/3/library/pathlib.html) (introduced with Python 3.4) instead of old Python 2 style unreadable `join` code :). No need to support Python 2.